### PR TITLE
Example of UserWithTweets entity with custom Put/Get/Delete resolvers

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
cc @Rainer-Lang, here is the example of an entity with references to another entities (I used  existent `User` and `Tweet` just to minimize new code and make sample app small and understandable, hope you will be able to map it onto your Person with Cars relationship!)

@nikitin-da PTAL :)

### CODE REVIEW

**Comment by nikitin-da** [comment]
Nice example and good start for tests =)

**Comment by artem-zinnatullin** [comment]
Oh shi :dancers: 

I forgot to add "not for merge yet" :)

@Rainer-Lang please feel free to review and comment this PR!

@nikitin-da sorry, my bad :)

**Comment by Rainer-Lang** [comment]
I don't think that this will work for me. 
I don't use such a container class UserWithTweets. I use only User and inside of the the user a list with Tweets.

**Comment by artem-zinnatullin** [comment]
Yep, in your case you can just eliminate User class and read it from the
cursor directly!

ср, 16 Сен 2015, 2:18, Rainer-Lang <notifications@github.com>:

> I don't think that this will work for me.
> I don't use such a container class UserWithTweets. I use only User and
> inside of the the user a list with Tweets.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/520#issuecomment-140578601>.
>
-- 

@artem_zin

**Comment by artem-zinnatullin** [comment]
The main idea was to show you how to retrieve Tweets of the User (Cars of
the Person) :)

I'll make another PR tomorrow with Persons and Cars but we won't merge it,
just for discussion :)

ср, 16 Сен 2015, 2:27, Artem Zinnatullin <artem.zinnatullin@gmail.com>:

> Yep, in your case you can just eliminate User class and read it from the
> cursor directly!
>
> ср, 16 Сен 2015, 2:18, Rainer-Lang <notifications@github.com>:
>
>> I don't think that this will work for me.
>> I don't use such a container class UserWithTweets. I use only User and
>> inside of the the user a list with Tweets.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/pushtorefresh/storio/pull/520#issuecomment-140578601>
>> .
>>
> --
>
> @artem_zin
>
-- 

@artem_zin

**Comment by Rainer-Lang** [comment]
Artem, I have some difficulties with the three resolvers. Could you please help me?

**Comment by Rainer-Lang** [comment]
Ah ok fine. :-)
Thanks a lot, Artem.

**Comment by artem-zinnatullin** [comment]
Sure, please ask your questions, I'll answer them after sleep :)

**Comment by Rainer-Lang** [comment]
Please make your pr with person and cars so we can discuss - without merge. ;-)
Good night.

File: storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/resolvers/UserWithTweetsGetResolver.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    @NonNull
+    private final GetResolver<User> userGetResolver;
+
+    // Sorry for this hack :(
+    // We will pass you an instance of StorIO
+    // into the mapFromCursor() in v2.0.0.
+    //
+    // At the moment, you can save this instance in performGet() and then null it at the end
+    @NonNull
+    private final ThreadLocal<StorIOSQLite> storIOSQLiteFromPerformGet = new ThreadLocal<StorIOSQLite>();
```
Comment (Rainer-Lang) [inline storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/resolvers/UserWithTweetsGetResolver.java]: Any plans when to release v2?

Comment (artem-zinnatullin) [inline storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/resolvers/UserWithTweetsGetResolver.java]: As soon as we will complete all issues with mileston v2.0.0 :) No concrete dates, but probably near to the end of this year, I hope :)

**Pull request merged**